orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/ClientSignatureValidationService.java

```diff
@@ -182,6 +182,10 @@ private PublicKey retrievePublicKey(ClientRegistry client, String kid)
             }
 
             JWK jwk = JWK.parse(unescapedPayload);
+            LOG.info(
+                    "Caching RP public signing key with key ID {} for client ID {}.",
```

Comment:
In general I prefer not having `.` on these as it looks like part of the value. The client ID should be attached to every log so it may be unnecessary anyway